config/locales/en.yml

```diff
@@ -305,19 +304,26 @@ en:
       other_agencies_count: 400+
       passport_fees: Passport fees
       popular: Popular on GOV.UK
+      promo_boxes:
+        text_1: Find out how the UK government is helping to reduce the HGV driver shortage.
+        text_2: Get information about vaccinations on NHS.UK.
+        text_3: Check what you need to do to travel internationally.
+        title_1: HGV driver shortage
+        title_2: COVID-19 vaccinations
+        title_3: 'Travel abroad: step by step'
       running_limited_company: Running a limited company
       see_all: Here you can see all
       services_and_information: Services and information
       statistics: statistics
       student_finance_login: Log in to student finance
       tax_account: Sign in to your personal tax account
-      travel_abroad: 'Travel abroad: step by step'
-      travel_abroad_check: Check what you need to do to travel internationally.
-      uk_bank_holidays: &uk_bank_holidays UK bank holidays
+      travel_abroad:
+      travel_abroad_check:
+      uk_bank_holidays: UK bank holidays
       uk_bank_holidays_dates_html: Check the dates for <a href="/bank-holidays" class="govuk-link" data-track-category="homepageClicked" data-track-action="promoTextLink" data-track-label="/bank-holidays" data-track-value="1" data-track-dimension="%{dimension}" data-track-dimension-index="29">bank holidays</a> in England, Wales, Scotland and Northern Ireland.
       universal_credit: Sign in to your Universal Credit account
-      vaccination: COVID-19 vaccinations
-      vaccination_description: Get information about vaccinations on NHS.UK.
+      vaccination:
+      vaccination_description:
```

Comment:
are we still referencing `vaccination` and `vaccination_description` somewhere (same question applies for `travel_abroad` and `travel_abroad_check`)?

Reply:
Thanks, looks like these got brought across from the Welsh translation file.  Have now removed.